packages/Molecules/Form/fields/fieldRplselect.vue

```diff
@@ -1,118 +1,384 @@
 <template>
-  <multiselect
-    :options="options"
-    :value="localValue"
-    v-bind="selectOptions"
-    :placeholder="schema.placeholder"
-    :custom-label="customLabel"
-    :max="schema.max || null"
-    :disabled="disabled"
-    @input="updateSelected"
-    @search-change="onSearchChange"
-    @tag="addTag"
-    :option-height="selectOptions.optionHeight"
-  >
-  </multiselect>
+  <div class="rpl-select" :class="{'rpl-select--open' : isOpen}">
+    <div v-if="$breakpoint.s" class="rpl-select__native">
+      <select :multiple="schema.multiselect" v-model="value">
```

Comment:
For mobile, the use of a label here (possibly part of the schema object?) would help with mobile screen readers.

Reply:
have added id on select element so it will bel labelled by the VFG generated label.